composer.json

```diff
@@ -27,6 +28,7 @@
         "symfony/phpunit-bridge": "^2.7 || ^3.0"
     },
     "provide": {
+        "ext-mongo": "*",
```

Comment:
> Maybe it will be needed to create branchs for `<php7` and `>=php7 `

We can't.

For me, the `provide` should be on `alcaeus/mongo-php-adapter`, not here because this is this library that provide a replacement for the deprecated extension.

BTW, this is the case: https://packagist.org/packages/alcaeus/mongo-php-adapter